format/src/main/scala/org/scalafmt/internal/TreePrinter.scala

```diff
@@ -472,7 +472,8 @@ object TreePrinter {
         dDef(t.mods, `def`, `this`, Nil, t.paramss, None, dbody)
       case m: Mod =>
         m match {
-          case t: Mod.Annot => `@` + print(t.init)
+          case t: Mod.Annot =>
+            `@` + SimpleTyp.wrap(t.init.tpe) + dArgss(t.init.argss)
```

Comment:
Can we keep this on a single like like the rest of the `Mod` cases

Reply:
`scalafmt` says nope